framework/src/utils/InputParameters.C

```diff
@@ -1049,6 +1057,23 @@ InputParameters::addParam<std::vector<MooseEnum>>(const std::string & /*name*/,
              "parameter is not required!");
 }
 
+template <>
+void
+InputParameters::addParam<MooseFunctorName>(const std::string & name,
+                                            const std::string & doc_string)
+{
+  addParamWithoutDefaultInner<MooseFunctorName>(name, appendFunctorDescription(doc_string));
+}
+
+template <>
+void
+InputParameters::addParam<MooseFunctorName, MooseFunctorName>(const std::string & name,
```

Comment:
what is the double < T , T > for?

Reply:
There are two template arguments for the `addParam` method that has a default argument, but I guess I don't need the second one (maybe it's deduced implicitly?). Anyway, moot point now with the new changes.